engine/engine.go

```diff
@@ -237,14 +248,21 @@ type Engine struct {
 }
 
 func (e *Engine) MakeInstantQuery(ctx context.Context, q storage.Queryable, opts *QueryOpts, qs string, ts time.Time) (promql.Query, error) {
+	span, ctx := tracing.StartSpanFromContext(ctx, "engine.MakeInstantQuery")
+	defer span.Finish()
```

Comment:
Do we expect the engine to take a lot of time before query execution phase? Does these spans add value?